src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,25 @@
-export const TodoInfo = () => {};
+import React from 'react';
+import getClassName from 'classnames';
```

Comment:
It's not critical, but the general naming for this lib is "cx" or just "classNames".
So you can do you import like this:
```
import cx from "classNames"
```